src/aero_kernel/src/userland/task.rs

```diff
@@ -282,6 +365,7 @@ impl Task {
             tid: pid.clone(),
             pid,
 
+            message_queue: MessageQueue::new(),
```

Comment:
When we make a child I believe it would be a nice idea to copy over the messages from the parent process?

Reply:
I don’t believe it would be a good idea since messages are addressed to a specific process